db/schema.sql

```diff
@@ -0,0 +1,92 @@
+-- TODO(brandon): check if deployed version of PostgreSQL has gen_random_uuid built in and remove pgcrypto extension if so
+CREATE EXTENSION pgcrypto; -- for gen_random_uuid()
+
+-- Identifies a particular VDAF.
+CREATE TYPE VDAF_IDENTIFIER AS ENUM(
+    'PRIO3',
+    'POPLAR1'
+);
+
+-- Corresponds to a PPM task, containing static data associated with the task.
+CREATE TABLE tasks(
+    id                     BYTEA PRIMARY KEY,         -- 32-byte TaskID as defined by the PPM specification
+    ord                    BIGINT NOT NULL,           -- the order of this aggregator for this task; 0 is leader, 1 or larger is helper
+    aggregator_endpoints   TEXT[] NOT NULL,           -- aggregator HTTPS endpoints, leader first
+    vdaf                   VDAF_IDENTIFIER NOT NULL,  -- the VDAF in use for this task
+    vdaf_verify_param      BYTEA NOT NULL,            -- the VDAF verify parameter (opaque message)
+    max_batch_lifetime     BIGINT NOT NULL,           -- the maximum number of times a given batch may be collected
+    min_batch_size         BIGINT NOT NULL,           -- the minimum number of reports in a batch to allow it to be collected
+    min_batch_duration     INTERVAL NOT NULL,         -- the duration of a single batch window
+    collector_hpke_config  BYTEA NOT NULL             -- the HPKE config of the helper (encoded HpkeConfig message)
+
+    -- TODO(brandon): include agg_auth_key if we decide it shouldn't go in a secret store
+);
+
+-- Individual reports received from clients.
+CREATE TABLE client_reports(
+    id            BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY, -- artificial ID, internal-only
+    task_id       BYTEA NOT NULL,      -- task ID the report is associated with
+    nonce_time    TIMESTAMP NOT NULL,  -- timestamp from nonce
+    nonce_rand    BIGINT NOT NULL,     -- random value from nonce
+    extensions    BYTEA NOT NULL,      -- sequence of encoded Extension messages
```

Comment:
Should this be `BYTEA[]`, since a report could contain multiple extensions?

Reply:
This is intended to be the entire serialized extensions vector; I updated the comment to clarify this.